.github/workflows/linter.yml

```diff
@@ -29,7 +29,7 @@ jobs:
           fetch-depth: 0
 
       - name: Lint Code Base
-        uses: super-linter/super-linter/slim@4e51915f4a812abf59fed160bb14595c0a38a9e7
+        uses: super-linter/super-linter/slim@88ea3923a7e1f89dd485d079f6eb5f5e8f937589
```

Comment:
I'm kind of confused. Why was it bumped here? It doesn't seem to be related to switching to Ubuntu 24.04.

Reply:
I'm guessing it fixed:
```
  proxy | 2024/06/05 07:32:38 [098] 200 https://github.com:443/super-linter/super-linter/git-upload-pack
updater | 2024/06/05 07:32:43 ERROR <job_837719984> Error processing super-linter/super-linter (Dependabot::SharedHelpers::HelperSubprocessFailed)
updater | 2024/06/05 07:32:43 ERROR <job_837719984> error: no such commit 4e51915f4a812abf59fed160bb14595c0a38a9e7
updater | 2024/06/05 07:32:43 ERROR <job_837719984> /home/dependabot/common/lib/dependabot/shared_helpers.rb:432:in `run_shell_command'
```
but it is unrelated to Ubuntu 24.04 anyway.